packages/web3-providers-ipc/src/index.ts

```diff
@@ -38,6 +35,9 @@ export default class IpcProvider<API extends Web3APISpec = EthExecutionAPI> exte
 > {
 	// Message handlers. Due to bounding of `this` and removing the listeners we have to keep it's reference.
 	protected _socketConnection?: Socket;
+	public get SocketConnection() {
```

Comment:
changelog update for this is required.

Reply:
Good catch! Thanks,
I just reverted this change and I will do it as part of #5887